utils/project/sync.go

```diff
@@ -196,7 +198,21 @@ func completeUpload(projectID string, files []string, modfiles []string, conURL
 	return resp.Status, resp.StatusCode
 }
 
-func ignoreFileOrDirectory(name string, isDir bool) bool {
+// Retrieve the ignoredPaths list from a .cw-settings file
+func retrieveIgnoredPathsListFromCWSettings(projectPath string) []string {
```

Comment:
This func name is rather long. Can this be shortened?

Reply:
Made it smaller